pyproject.toml

```diff
@@ -274,5 +274,7 @@ filterwarnings = [
     # https://docs.pytest.org/en/latest/how-to/failures.html#warning-about-unraisable-exceptions-and-unhandled-thread-exceptions
     "error::pytest.PytestUnraisableExceptionWarning",
     "error::pytest.PytestUnhandledThreadExceptionWarning",
+    # Remove the following when this issue is fixed: https://github.com/CycloneDX/cyclonedx-python-lib/issues/870
+    "ignore::DeprecationWarning:cyclonedx.model.tool",
```

Comment:
I wonder if we could temporary ignore the warning using a context manager - https://docs.python.org/3/library/warnings.html#temporarily-suppressing-warnings - instead of filtering all warning?